isb_lib/authorization/orcid/__init__.py

```diff
@@ -1,12 +1,22 @@
 import logging
+from typing import Optional
 
 import requests
 from requests import Response
+from isb_web import config
 
 ORCID_API_CONTENT_TYPE = "application/orcid+json"
 ORCID_API_VERSION = "2.1"
 ORCID_API_ENDPOINT = "record"
-ORCID_API_HOSTNAME = "pub.sandbox.orcid.org"
+ORCID_API_HOSTNAME = config.Settings().orcid_hostname
+ORCID_TOKEN_REDIRECT_URI = config.Settings().orcid_token_redirect_uri
+ORCID_CLIENT_ID = config.Settings().orcid_client_id
+ORCID_CLIENT_SECRET = config.Settings().orcid_client_secret
```

Comment:
We are going to need to figure out how to store these properly in Docker so that they are available securely.  Checking them into a config file in plaintext is not the right way to do this.

Reply:
Though, once they're in the Docker keystore, this code should continue to function as we'll just be using that to set the environment variables.